src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -796,6 +805,10 @@ private void WriteField(Field field)
                         WriteLine("private readonly {0} _{1};", field.Type, field.Name);
                         WriteLine("public override {0}{1} {2} {{ get {{ return _{2};}} }}", (IsNew(field) ? "new " : ""), field.Type, field.Name);
                     }
+                    else if (field.Override)
+                    {
+                        WriteLine($"public new {field.Type} {field.Name} => base.{field.Name}!;");
```

Comment:
>! [](start = 93, length = 1)

Why the `!`?
Consider leaving a comment

Reply:
The point of overriding a property is to change its nullability, usually from nullable to non-nullable. The base would be annotated as nullable, but since the base property is always set via the `base` call in the constructor, as long as the parameter to the subtype's constructor is not nullable, this is always safe (but not trackable).

I'll add this in a comment.